api/controllers/latest.js

```diff
@@ -67,7 +68,7 @@ if (process.env.USE_ATHENA) {
         parameter: r.parameter,
         value: Number(r.value),
         unit: r.unit,
-        date_utc: r.date.utc,
+        date_utc: hiveParse(r.date).utc,
```

Comment:
I might be missing it, but where is `hiveParse()` coming from?

Reply:
hiveParse is here https://github.com/openaq/openaq-api/blob/develop/lib/utils.js#L232